jetty9-compat-base/src/main/java/com/google/apphosting/runtime/jetty9/SessionManager.java

```diff
@@ -362,9 +362,10 @@ static String lastId() {
    * Constructs a SessionManager
    *
    * @param sessionStoresInWriteOrder The SessionStores in the order to which they should be
-   *        written. When attempting to load a session, the read order is the opposite of the write
-   *        order, so if the write order is A, B, C, when reading we will first consult C, and if
-   *        the session is not found move on to B, and if not found then on to A.
+   *                                  written. When attempting to load a session, the read order is
```

Comment:
That change wasn't in my IDE history nor was it repeated either by an IDE format or a googlestyle format, at least not from any starting point I tried.  Mystery?   Fixing now.